client/src/components/Reports/index.js

```diff
@@ -1,16 +1,179 @@
+ },
+ title:{
+   color:red[900]
+ },
+ head:{
+   backgroundColor:red[200],
+   color: theme.palette.common.red,
+ },
+ primary: { main: red[500] },
+ formControl: {
+    margin: theme.spacing.unit,
+    minWidth: 120,
+    marginLeft:"20%"
+  },
+  selectEmpty: {
+    marginTop: theme.spacing.unit * 2,
+  },
+});
+
+class Reports extends React.Component {
+  state = {
+   month: new Date().getMonth() + 1,
+   year:new Date().getFullYear(),
+   labelWidth: 0,
+   carLogs:null
+ };
+
+  componentDidMount() {
+    this.getLogs();
+  }
+  handleChange = name => event => {
+   this.setState({ [name]: event.target.value },()=>{
+     this.getLogs();
+   });
+ };
+ getLogs = () => {
+   getData(`/logs/${this.state.year}/${this.state.month}`).then(carLogs => {
+     this.setState({ carLogs });
+   });
+ };
+
+  download=()=>{
+    console.log('year',this.state.year,'month',this.state.month);
+    fetch(`/logmonth/${this.state.year}/${this.state.month}`).then(response=>{
+      return response.blob();
+    }).then(blob=>{
+      download(blob,"logs.xlsx")
+    })
+  }
   render() {
+    const { classes } = this.props;
+    const { carLogs } = this.state;
     return (
-      <React.Fragment>
-        <Add />
-        <Show />
-      </React.Fragment>
-    );
+      <div>
+      <h1 align="center" className={classes.title}>Vehicle Log Sheet</h1>
+
+      <FormControl className={classes.formControl}>
+          <InputLabel htmlFor="uncontrolled-native">Month</InputLabel>
+          <NativeSelect defaultValue={this.state.month}
+          input={<Input name="month"
+          id="uncontrolled-native"
+          onChange={this.handleChange('month')}
+           />}>
+            <option value="" />
+            <option value={1}>1</option>
+            <option value={2}>2</option>
+            <option value={3}>3</option>
+            <option value={4}>4</option>
+            <option value={5}>5</option>
+            <option value={6}>6</option>
+            <option value={7}>7</option>
+            <option value={8}>8</option>
+            <option value={9}>9</option>
+            <option value={10}>10</option>
+            <option value={11}>11</option>
+            <option value={12}>12</option>
+          </NativeSelect>
+          <FormHelperText>Select Date to see reports on it</FormHelperText>
+        </FormControl>
+        <FormControl className={classes.formControl}>
+          <InputLabel htmlFor="uncontrolled-native">Year</InputLabel>
+          <NativeSelect
+            className={classes.selectEmpty}
+            name="year"
+            defaultValue={this.state.year}
+            onChange={this.handleChange('year')}
+          >
+            <option value="" >
+            </option>
+            <option value={2019}>2019</option>
+            <option value={2020}>2020</option>
+            <option value={2021}>2021</option>
+            <option value={2022}>2022</option>
+            <option value={2023}>2023</option>
```

Comment:
you can apply same principle as the comment above

Reply:
Done :+1: